sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -754,6 +826,12 @@ TEST_F(DifUnavailableError, ReadIV) {
   EXPECT_EQ(dif_aes_read_iv(&aes_, &iv), kDifUnavailable);
 }
 
+TEST_F(DifUnavailableError, ProcessData) {
+  dif_aes_data_t data[2] = {}, out[2] = {};
```

Comment:
Don't declare multiple variables on the same line if they have non-trivial declarators (like arrays).